### PR TITLE
Closes #1 - Create initial index.html markup

### LINKED ISSUE #1: Create initial index.html markup

### PR DESCRIPTION
**Pull request by koldovsky**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+
+	<head>
+	    <meta charset="UTF-8">
+	    <meta name="viewport" content="width=device-width, 
+	initial-scale=1.0">
+	    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+	    <title></title>
```
Comment (AleshkaGvozd) [inline index.html]: title should not be empty

**Review by AleshkaGvozd — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Halynachervoniak) [inline index.html]: Title shoudn't be empty

**Review by Halynachervoniak — CHANGES_REQUESTED** [review]

**Review by AnnaParashchak — CHANGES_REQUESTED** [review]
Title should not be empty

**Review by DannyBoj — CHANGES_REQUESTED** [review]
Title should not be empty

**Review by ArturKhomenkoo — CHANGES_REQUESTED** [review]
Title shouldn`t be empty

**Review by ArturKhomenkoo — APPROVED** [review]

**Review by AleshkaGvozd — APPROVED** [review]

**Review by AnnaParashchak — APPROVED** [review]

**Review by DannyBoj — APPROVED** [review]

**Review by Halynachervoniak — APPROVED** [review]

**Pull request merged**